src/colorizer/types.ts

```diff
@@ -76,6 +76,18 @@ export const featureTypeSpecs: { [T in FeatureDataType]: FeatureTypeSpec<T> } =
   },
 };
 
+export type FrameLoadResult = {
+  frame: number;
+  /** False if frame loading encountered an error. */
+  isFrameLoaded: boolean;
+  backdropKey: string | null;
+  /**
+   * False if backdrop loading encountered an error. Note that this will be
+   * true if the backdrop is not visible.
+   */
+  isBackdropLoaded: boolean;
```

Comment:
I guess a better name for this would be, `didBackdropLoadEncounterError` but it's not as concise. Any thoughts?

Reply:
`backdropError`.  And maybe change the other one to `frameError`.   Then you can write code like `if (frameResult.backdropError || frameResult.frameError) ...`
You could even keep extended error info in here if needed.